tests/tekton-resources/tasks/teardown/awscli-eks.yaml

```diff
@@ -70,6 +70,166 @@ spec:
           echo "Role $PIA_ROLE_NAME does not exist, no action needed."
         fi
       done
+  - name: delete-karpenter-role
```

Comment:
lets have a separate tear down for ultra clusters to ensure it won't break exisitng pipelines when there is a bug.